Cog.xcodeproj/project.pbxproj

```diff
@@ -2654,6 +2680,7 @@
 				SWIFT_OBJC_BRIDGING_HEADER = "Cog-Bridging-Header.h";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
+				USER_HEADER_SEARCH_PATHS = Preferences/Preferences;
```

Comment:
I don't know yet a good way to include `Shortcuts.h` in Cog target.
@kode54 Can you help me?

Reply:
When you use normal header maps in a project or sub-project, simply including the header file somewhere in the file tree for the project is enough to be able to:

```
#import <someheader.h>
```

This is what I do for the headers I include in each of the plugins.

Reply:
`File not found` for me

Reply:
Oh, crap. I don't know then. Somehow it works for the headers in each of the plugin projects. Maybe the project you're trying to use it in has header maps turned off? I know I had to do that for some frameworks due to weird issues like bonehead Xcode seeing a project header with the same name as a system header, and resolving the project header for:

```
#include <header.h>
```

Even when it's included that way within the identically named project header file, which is included elsewhere using:

```
#include "header.h"
```